INTHEWILD.md

```diff
@@ -291,6 +291,7 @@ Currently, **officially** using Airflow:
 1. [REA Group](https://www.rea-group.com/)
 1. [Rakuten](https://www.rakuten.com)
 1. [Rapido](https://rapido.bike/) [[@ChethanUK](https://github.com/ChethanUK)]
+1. [Raisin](https://www.raisin.com/) [[@davido912](https://github.com/davido912)]
```

Comment:
Can you move this to line 292, since we order this list alphabetically, which is why the static check is failing

Reply:
@kaxil did you by any chance just make changes on the master? I swear I put it in the correct order alphabetically and not under Rapido, haha. Will try to rebase

Reply:
Haha I swear, I didn't 😂😀

Reply:
Works now 💯